src/types/misc.ts

```diff
@@ -183,6 +183,7 @@ export type FunctionReturnValue = string | number | boolean;
 
 export interface ClipboardCell {
   cell?: Cell;
+  style?: Style;
```

Comment:
Personal opinion, but I hate when I copy something from discord or vs code, paste it somewhere (e.g. a task pad) and the style is also copied.

The first thing I always do it to fight with the editor to remove the style. Maybe it's less annoying with tables

Reply:
I 100% agree with you. That being said, I feel like it make more sense for tables, since we might have different styles for headers/highligted cells ... 
Might be worth to ask around.

Reply:
I agree that it's a pita but in the end, it can still be useful in some cases. Mamu made the good repark that trl Shift v actually pastes the text/plain and not the text/html content, don't recall where he foundthe invormation but it looks damn true